components/script/body.rs

```diff
@@ -166,6 +176,26 @@ fn run_form_data_algorithm(root: &GlobalScope, bytes: Vec<u8>, mime: &[u8]) -> F
     }
 }
 
+#[allow(unsafe_code)]
+fn run_array_buffer_data_algorithm(cx: *mut JSContext, bytes: Vec<u8>) -> Fallible<FetchedData> {
```

Comment:
This function should be marked unsafe, since it accepts a raw pointer.

Reply:
I've specified unsafe scope [here](https://github.com/servo/servo/pull/20406/files#diff-ae0ff1fd98b06dfb13baf427cdffc28aR182) as similar to other [places](https://github.com/servo/servo/pull/20406/files#diff-ae0ff1fd98b06dfb13baf427cdffc28aR129) - mainly cause specifying fn itself as unsafe will raises compilation error as unsafe invocation is propagated in line 116.